spec/amp-cache-transform.md

```diff
@@ -163,12 +201,17 @@ the latter response.
 If the proxy can ensure that a cached response satisfies a new request, then it
 can serve that response. It can do that by comparing the `AMP-Cache-Transform`
 response header of the cached response to the `AMP-Cache-Transform` request
-header of the new request:
-
- 1. If the request includes an unparameterised `any` identifier and the response
-    includes a `AMP-Cache-Transform` header, then it satisfies the request.
- 2. If the response identifier is included in the request list, and
-    unparameterised in both cases, then it satisfies the request.
+header of the new request. For instance, the following algorithm determines if a
+response satisfies a request:
+
+ 1. If the response matches any parameterised identifier in the request list,
+    per:
+   1. If its identifier is not `any` and the response's identifier is not
+      identical, then the response doesn't match.
+   2. If it includes a `v` parameter and the response's `v` parameter is either
+      missing or not an element of it, then the response doesn't match.
```

Comment:
s/it/the allowed v parameter values/

There are two "it"s in this sentence which are intended to dereference to two different concepts.